internal/worker_runner.go

```diff
@@ -114,12 +110,8 @@ func (wr *workerRunner) Run() {
 				}
 				// query OBD signals
 				wr.queryOBD()
-				isLastVoltageOk = true
 			} else {
-				if isLastVoltageOk {
-					wr.logger.Info().Msgf("voltage not enough to query obd : %.1f", powerStatus.VoltageFound)
-					isLastVoltageOk = false
-				}
+				wr.logger.Info().Ctx(context.WithValue(context.Background(), "threshold", 10)).Msgf("voltage not enough to query obd : %.1f", powerStatus.VoltageFound)
```

Comment:
This is an example how we can simplify "isLastVoltageOk" login logic using FilterHook